### PR TITLE
Improvements to Portuguese Brazilian translation

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -583,12 +583,12 @@
   <br/><br/>
   É um software livre de código aberto, desenvolvido pela <a href="https://phoenix.acinq.co">ACINQ</a> sob a <a href="https://www.apache.org/licenses/LICENSE-2.0.txt">Licença Apache 2.0</a>.
   <br/><br/>
-  Observe que esta carteira é não custodial: <strong>significa que somente você tem a custódia das 12-palavras da carteira, também conhecida como seed</strong>. Não forneça a seed para ninguém, e tome cuidado com ataques phishing.
+  Observe que esta carteira é não custodiante: <strong>significa que somente você tem a custódia das 12 palavras da carteira, também conhecida como seed</strong>. Não forneça a seed para ninguém, e tome cuidado com ataques phishing.
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:586]:
```suggestion
  Observe que esta carteira é não custodiante: <strong>significa que somente você tem acesso as 12 palavras da carteira, também conhecida como seed</strong>. Não forneça a seed para ninguém, e tome cuidado com ataques phishing.
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -609,8 +609,8 @@
   <string name="paymentsettings_defaultdesc_dialog_description">As solicitações de pagamento que você criar irão conter esta descrição. Você pode substituí-las caso a caso.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:609]:
```suggestion
  <string name="paymentsettings_defaultdesc_dialog_description">As cobranças que você gerar irão conter esta descrição. Você pode substituí-las caso a caso.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -583,12 +583,12 @@
   <br/><br/>
   É um software livre de código aberto, desenvolvido pela <a href="https://phoenix.acinq.co">ACINQ</a> sob a <a href="https://www.apache.org/licenses/LICENSE-2.0.txt">Licença Apache 2.0</a>.
   <br/><br/>
-  Observe que esta carteira é não custodial: <strong>significa que somente você tem a custódia das 12-palavras da carteira, também conhecida como seed</strong>. Não forneça a seed para ninguém, e tome cuidado com ataques phishing.
+  Observe que esta carteira é não custodiante: <strong>significa que somente você tem acesso as 12 palavras da carteira, também conhecida como seed</strong>. Não forneça a seed para ninguém, e tome cuidado com ataques phishing.
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:586]:
```suggestion
  Observe que esta carteira é do tipo não custodiante: <strong>significa que somente você tem acesso as 12 palavras da carteira, também conhecida como seed</strong>. Não forneça a seed para ninguém, e tome cuidado com ataques phishing.
```

**Review by dpad85 — APPROVED** [review]
👍 Thanks!

**Pull request merged**